### ISSUE TITLE
Clicking on a matched error re-adds cwd

### ISSUE THREAD

**Issue by jhasse** [noseglid/atom-build#194]
I have set my cwd to `build` which works fine and matches the errors. But when I click on the error in the build panel atom-build can't the file. After clicking several times the error message looks like this:

```
Matched file does not exist: build//build//build//build//build//../src/main.cpp
```

So it seems that my cwd (which is set to `build/`) gets added multiple times.

**Comment by noseglid** [comment]
Could you prived a custom build file and some sample output which produces this error?

**Comment by jhasse** [comment]
```bash
mkdir test
cd test
echo "error" > main.c
echo '{"cmd":"gcc ../main.c","cwd":"build","errorMatch":"(?<file>[^: \\n]+):(?<line>\\d+):(?<col>\\d+): "}' > .atom-build.json
mkdir build
atom .
```

Trigger a build and click multiple times on the error.

**Comment by noseglid** [comment]
Thanks for the very detailed information! I have located the issue and have a fix for it. It will be included in the next version `v0.48.0`. If you want to test it now, the code is available on the branch `error-match-prepend-cwd`.

**Comment by jhasse** [comment]
I'll wait. Thanks for being so fast with the fix :)
